pkg/procmatch/cmd/procmatch/procmatch.go

```diff
@@ -0,0 +1,13 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/DataDog/datadog-agent/pkg/procmatch"
+)
+
+func main() {
+	fmt.Println(procmatch.Match(strings.Join(os.Args[1:], " ")))
```

Comment:
Do we actually need a new command bundled with the agent? If we want it for testing then I'd rather make it some sub-command of the `agent` binary itself and have it support a bit more (show me all detected integrations on my current host, ...)